bears/general/KeywordBear.py

```diff
@@ -79,22 +79,55 @@ def run(self,
             filename,
             file,
             keywords: list=['todo', 'fixme'],
+            regex_keyword: str='',
             dependency_results: dict=None):
         '''
         Checks the code files for given keywords.
 
         :param keywords:
             A list of keywords to search for (case insensitive).
             Default are TODO and FIXME.
+        :param regex_keyword:
+            A regular expression to search for matching keywords in a file.
         '''
         keywords_regex = re.compile(
```

Comment:
changed keywords_regex to simple_keywords_regex
Now ```regex``` is for ```regex_keyword``` setting and ```simple_keywords_regex``` is for ```keywords``` setting